lib/AppleNote.rb

```diff
@@ -508,7 +520,7 @@ def generate_html
       }
     end
 
-    @html = builder.doc.root
+    builder.doc.root
```

Comment:
I'm curious why you removed the caching. This function is called in multiple places (the HTML doesn't just go on disk, it also is put into the JSON for example) and the idea was to build the HTML once and return it every other time quickly.

Reply:
I had removed the caching mostly because I was thinking of a scenario where these methods could be called with different values for `individual_files` — of course that can't happen within the same run at the moment. However, I think I can restore the caching and still make it work if there was a need to generate both versions (`individual_files` on and off) within the same run.